src/select.scss

```diff
@@ -0,0 +1,81 @@
+@import "./settings";
+@import "./mixins";
+@import "./functions";
+
+$block: #{$fd-namespace}-select;
+
+.#{$block} {
+  @include fd-reset();
+
+  display: block;
+
+  &__control {
+    @include fd-form-text();
+    @include fd-var-size("height", $fd-forms-height, --fd-forms-height);
+
+    width: 100%;
+    text-align: left;
+
+    @include fd-icon-base("after");
+
+    @include fd-icon-glyph("slim-arrow-down","after") {
+      transition: transform $fd-animation--speed ease-in;
+    }
+
+    position: relative;
+    padding-right: $fd-forms-height + $fd-forms-padding - 2;
+
+    &::after {
+      @include fd-var-color("color", $fd-color--action, --fd-color-action-1);
```

Comment:
We don't need to use the `fd-var-color` mixins anymore - you can put `var(--fd-color-action-1)` and the postcss plugins will take care of the rest :)